modules/ROOT/pages/_partials/include-insights-charts.adoc

```diff
@@ -0,0 +1,386 @@
+Top Slowest Requests by Path::
+Up to 20 endpoints with the slowest requests
++
+The requests must take more than 25ms. If no data is available, then no requests take more than 25ms to process.
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+
+Request Latency by Endpoint::
+Inbound or outbound latency, in milliseconds, by endpoint
++
+The chart captures the average latency and the latency value of requests at the p50, p75, p90, p95, and p99 percentiles.
+ 
+//end::app-detail-requests[]
+//-----------------------//
+//
+//-----------------------//
+//  APP DETAIL ERRORS: Errors tab
+//tag::app-detail-errors[]
+Failed Outbound Request Volume::
+Number of failed outbound requests at specific time points
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+
+Top 20 Failed Requests by Total Count::
+Number of inbound or outbound requests to the 20 endpoints with the most failed requests
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+
+Top 20 Failed Outbound Requests by Total Count::
+Number of unsuccessful outbound requests to the 20 endpoints with the most failed requests
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+//end::app-detail-errors[]
+//-----------------------//
+//
+//-----------------------//
+// APP DETAIL JVM: JVM tab
+//tag::app-detail-jvm[]
+//TODO: Old Gen GC Count -> Old Generation Garbage Collection Count (SATHISH)
+Old Gen GC Count::
+Number of times old generation Java garbage collection occurred
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+
+//TODO: Old Gen GC Duration -> Old Generation Garbage Collection Duration (SATHISH)
+Old Gen GC Duration::
+Amount of time taken for old generation Java garbage collection
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+
+//TODO: Young Gen GC Count -> Young Generation Garbage Collection Count (SATHISH)
+Young Gen GC Count::
+Number of times young generation Java garbage collection occurred
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+
+//TODO: Young Gen GC Duration -> Young Generation Garbage Collection Duration (SATHISH)
+Young Gen GC Duration::
+Amount of time taken for young generation Java garbage collection
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+
+Heap Committed::
+Amount of physical memory committed to the JVM
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+
+Heap Usage::
+Amount of physical memory used
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+
+Thread Count::
+//TODO: QUESTION: threads that are available or used??? (SATHISH)
+Number of threads for simultaneous requests
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+
+Committed Virtual Memory::
+Amount of memory (in MB) committed to the JVM
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+
+CPU Utilization::
+CPU work as a percentage of total CPU time
```

Comment:
I can't quite put a finger on this one, but is this description accurate?

Reply:
@vimil-saju